public/js/index.js

```diff
@@ -1,3 +1,5 @@
 function displayMessage() {
     return 'Hello, World!';
```

Comment:
I know this method was written by me, but that was before we had the style guide.
Could you correct the indentation please?